IdokladSdk/Models/RecurringInvoice/Patch/InvoiceItemTemplatePatchModel.cs

```diff
@@ -21,6 +22,8 @@ public class InvoiceItemTemplatePatchModel : ValidatableModel
         /// <summary>
         /// Gets or sets item name.
         /// </summary>
+        [Required(AllowEmptyStrings = false)]
```

Comment:
V patch modelu nemůžeme takto použít required. Tímto bychom docílili toho že při patchi nebude možné upravit jen jednu property, ale byly by povinné i všechny required. V patchi by měl být reuired pouze Id. Tady ti šlo pravděpodobně o to aby se nepovolovaly prázdné stringy a na to v patchi používáme atribut [NotEmptyString]

Reply:
Ne, primárně nám jde o to, že souce-generátor spopéhá na `[Required]` atribut, aby následně v aplikaci byl u těchto polí na Views symbol 'hvězdičky' + a tímpádem následná validace končila chybou 'Toto pole je vyžadováno'. Nyní není žádná validace na tomto poli